workers/exclusive-content/LICENSE_MIT

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2020 Sabine Schaller <sabine@coil.com>
```

Comment:
We usually put stuff under the Apache-2.0 license. Might be a requirement of the WICG but in any case probably best to use the same license we use elsewhere in this project

Reply:
No problem, I'll change that. I think this was the default.